docs/src/prototypes/chatMessages/ControlMessages/GroupControlMessages.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import * as keyboardKey from 'keyboard-key'
+
+import { List, chatBehavior, Accessibility, Flex, Icon } from '@stardust-ui/react'
+import ControlMessage from './ControlMessage'
+
+const controlMessagesGroupBehavior: Accessibility<any> = props => {
+  const behaviorData = chatBehavior(props)
+
+  behaviorData.attributes.root = {
+    ...behaviorData.attributes.root,
+    'data-is-focusable': true,
+  }
+
+  behaviorData.focusZone = {
+    mode: behaviorData.focusZone.mode,
+    props: {
+      ...behaviorData.focusZone.props,
+      shouldFocusOnMount: true,
+      shouldFocusInnerElementWhenReceivedFocus: true,
+      defaultTabbableElement: undefined,
+    },
+  }
+  return behaviorData
+}
+
+const GroupControlMessages = () => {
```

Comment:
Can we have here props with items param, and we can use this component in a more realistic way. We can extract the items in a separate file, called mockData or something similar.